avocado/plugins/list.py

```diff
@@ -101,10 +107,19 @@ def _display(self, test_matrix, stats):
             LOG_UI.debug(line)
 
         if self.args.verbose:
-            LOG_UI.debug("")
+            LOG_UI.info("")
+            LOG_UI.info("TEST TYPES SUMMARY")
+            LOG_UI.info("==================")
             for key in sorted(stats):
                 LOG_UI.info("%s: %s", key.upper(), stats[key])
 
+            if tag_stats:
```

Comment:
Not sure whether this section should be showed only when there are tags. We always show `TEST TYPES SUMMARY` so I'd slightly prefer showing this always. What do you think?